x/stake/handler.go

```diff
@@ -40,19 +43,42 @@ func NewHandler(k keeper.Keeper, govKeeper gov.Keeper) sdk.Handler {
 			return handleMsgUndelegate(ctx, msg, k)
 		// case MsgSideChain
 		case types.MsgCreateSideChainValidator:
+			if sdk.IsUpgrade(sdk.FirstSunsetFork) {
+				return sdk.ErrMsgNotSupported("").Result()
+			}
 			return handleMsgCreateSideChainValidator(ctx, msg, k)
 		case types.MsgEditSideChainValidator:
```

Comment:
Why `MsgEditSideChainValidator` is not disabled like `MsgEditSideChainValidatorWithVoteAddr`?

Reply:
accept

Reply:
`MsgEditSideChainValidator ` was disable by BEP159, but `BEP159` is not enabled on mainnet. so I think we should disable it.

Reply:
<img width="888" alt="image" src="https://github.com/bnb-chain/bnc-cosmos-sdk/assets/25412254/b0cf334a-c0fe-40b0-8c34-d651c6cc8dca">
`SideConsAddr` edition was allowed by BEP159, use can only change consAddr on testnet.